extensions/Prefixes.md

```diff
@@ -51,6 +51,7 @@ Once a prefix is approved, it should be added to the [glTF Validator's list of k
 | `ICOSA` | Icosa Foundation | https://icosa.foundation/ | [#2208](https://github.com/KhronosGroup/glTF/issues/2208) |
 | `INTEL` | Intel Corporation | https://www.intel.com/ | [#2142](https://github.com/KhronosGroup/glTF/issues/2142) |
 | `KDAB` | The KDAB Group | https://www.kdab.com/ | [#1728](https://github.com/KhronosGroup/glTF/pull/1728) |
+| `KITTYCAD` | KittyCAD Inc. | https://kittycad.io/<br>`david at kittycad.io` | [#2344](https://github.com/KhronosGroup/glTF/issues/2344) |
```

Comment:
@alteous please confirm if the name should have a comma before "Inc." since the website has both spellings.

Reply:
Thanks for checking, the company name is filed with a comma, so "KittyCAD, Inc." would be preferable.